screeps-game-api/src/objects/impls/flag.rs

```diff
@@ -1,10 +1,10 @@
-use stdweb::Value;
-
-use {
+use crate::{
     constants::{Color, ReturnCode},
+    macros::*,
     objects::{Flag, HasPosition},
     traits::TryFrom,
 };
+use stdweb::Value;
```

Comment:
I could change them, but `rustfmt` places them at the top.

super -> crate -> external crate. Do you feel the need to change that?

Reply:
It's just a different formatting than what the repository had since the last refactoring, and what's currently in https://github.com/daboross/screeps-in-rust-via-wasm/blob/master/CONTRIBUTING.md#use-formatting?

As I understand it, `rustfmt` just orders everything alphabetically, and doesn't explicitly make a choice to put them in that order.  [fmt-rfcs issue 131](https://github.com/rust-dev-tools/fmt-rfcs/issues/131) for getting it to do that has been open forever, but unless I'm running on outdated information that was never resolved, and the current behavior is by chance (it does stuff like order `std::` alphabetically in between other external crate imports).